spec/models/reservation_spec.rb

```diff
@@ -1023,13 +1016,19 @@
   end
 
   describe '#start_reservation!' do
-    it "sets actual start time", :timecop_freeze do
+    it "sets actual start time", :time_travel do
       reservation.start_reservation!
       expect(reservation.actual_start_at).to eq(Time.current)
     end
 
     context "with a running reservation" do
-      let!(:running) { create :setup_reservation, product: instrument, reserve_start_at: 1.hour.ago, reserve_end_at: Time.current, actual_start_at: 1.hour.ago }
+      let!(:running) do
+        start_time = 1.hour.ago - 1.second
```

Comment:
We didn't need it because the clock used to move. There could be a clearer way to do this, but the problem came down to `OrderDetail#has_completed_reservation?` being `false` because `reserve_end_at` had not passed, and never will.